cypress/integration/tabs.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Tabs component
+  I want to change Table component properties
+
+  Background: Open Tabs component page
+    Given I open "Tabs" component page
+
+  @positive
+  Scenario Outline: I align Tabs to <direction>
+    When I select align to "<direction>"
+    Then Tabs component is align to "<value>"
+    Examples:
+      | direction | value |
+      | right     | right |
+      | left      | start |
```

Comment:
if the values on which we make an assertion are not logical, I mean - left -> start, maybe we should put values into steps ? And pass only the direction ?

Reply:
moving that to step make ifology inside step code, in my opinion it is better to pass value